src/utils/constants.js

```diff
@@ -20,3 +21,10 @@ export const SKIP_KEYWORDS = [
   'oneOf',
   'not',
 ];
+
+/**
+ * Keywords used in schema that are descriptors.
+ * These are typically displayed in the rows of the right panel,
+ * separated by a blank line from the displayed chips.
+ */
+export const DESCRIPTOR_KEYWORDS = ['description', 'title'];

```

Comment:
Included the `title` keyword to be separated along with the `description` keyword since it seemed better UI to only display keywords related to specifications in chip form and leave descriptive keywords out in simple text.
So the current display is like:

![image](https://user-images.githubusercontent.com/29671309/71973237-c8279b00-3251-11ea-895b-75dc7cadf20d.png)

I was also thinking of maybe removing the `title: ` and `description: ` literals to further clean up the UI as well. Maybe in a format of `(title) description` ?

<img src="https://user-images.githubusercontent.com/29671309/71973588-a7ac1080-3252-11ea-878c-198b5918e9e8.png" width="500px" />

What do you think about this idea?

Reply:
> I was also thinking of maybe removing the `title:` and `description:` literals to further clean up the UI as well

Agreed.

>.  Maybe in a format of (title) description ?

You can definitely give that a try. Another idea that comes to mind is to perhaps put the title and description in their own line and then use a different font-weight (e.g., bold) for the title. Something like:

---
(chip-1) (chip-2) ...

<strong>title</strong>
description

Reply:
> You can definitely give that a try. Another idea that comes to mind is to perhaps put the title and description in their own line and then use a different font-weight (e.g., bold) for the title. Something like:

Implemented bold weight to titles!